### PR TITLE
Added summary of chen 2020 LIFF paper

### PR DESCRIPTION
**Pull request by trowk**

### CODE REVIEW

File: _summaries/chen2020learning_1.md [hunk 1]
```diff
@@ -3,17 +3,97 @@ layout: summary
...
+x4. The authors also attempted to generalize results beyond the downsampling
+scales provided, generating downsampling scales of x6, x12, x18, x24, and x30
+independently. They used the EDSR-baseline and RDN models as the encoders
+($$E_\varphi$$) and used a 5-layer MLP with RELU activation and hidden layer
+sizes of 256 as the decoder function.
+
+On the standard, in-distribution images, LIFF has competitive results. It does
+not noticeably outperform the other models that are tested on the dataset.
+However, on the models that were downsampled beyond the standard dataset, LIFF
+did have noticeably higher performance.
```
Comment (sritank) [inline _summaries/chen2020learning_1.md:74]: I know the paper did not show any image results in the experiment section and only a table. However, I think it would be good to show a visual comparison to other models, like the one shown in Figure 5

**Review by sritank — CHANGES_REQUESTED** [review]
Overall great summary! You captured all the details in the paper. Only minor comments on what to add/change

File: _summaries/chen2020learning_1.md [hunk 2]
```diff
@@ -3,17 +3,97 @@ layout: summary
...
+performance.
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Local Implicit Image Function (LIIF) attempts to provide a continuous
+representation for images.
+* LIFF has an encoder that maps images to a matrix of latent codes
+* LIFF uses a MLP decoder function to map coordinates and latent codes to rgb
```
[the remark this answers is not in the document]
Comment (trowk) [reply _summaries/chen2020learning_1.md]: Mentioned the resolution results for LIFF instead

**Review by sritank — APPROVED** [review]

**Comment by sritank** [comment]
Looks good to me
